acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/IbftPrivacyClusterAcceptanceTest.java

```diff
@@ -0,0 +1,511 @@
+/*
```

Comment:
This looks like a copy of PrivacyClusterAcceptanceTest.java.  Can/should the common bits be abstracted instead of duplicated?

Reply:
Additionally,  look at PR 1483 as I think a bunch of PrivacyClusterAcceptanceTest got rewritten

Reply:
yes - this PR is no longer relevant and can be closed. I don't have permission to do so.